extensions/reviewed/FireBullet.json

```
@@ -1,15 +1,15 @@
 {
   "author": "@4ian",
-  "category": "Movement",
-  "description": "Allow the object to fire bullets with customizable speed, angle and fire rate.\n\nCooldown: Time between shots (seconds)\nBullet Quantity: Number of bullets shot at once\nAngle Variance: Each bullet trajectory will be adjusted by a random value within this range (degrees)\nFiring Arc: Range of angles (in degrees) that bullets will shoot. Bullets are evenly spread within this range.\nRotate bullet: Change angle of bullet to match the direction it is travelling (enabled by default)\n",
+  "category": "",
```

```suggestion
  "category": "Movement",
```
"Game mechanics" could work too.